plugin.yaml

```diff
@@ -1,4 +1,4 @@
-name: "push"
+name: "legacy-push"
 version: "0.9.0"
```

Comment:
is the version increased?

Reply:
Version numbers are not bumped until the tag is being cut. The process of updating the version number is separate from introducing the change. That way multiple PRs do not introduce merge collisions.